Makefile

```diff
@@ -1,26 +1,40 @@
 DC := $(shell which docker-compose)
+HOSTUSER := $(shell id -u):$(shell id -g)
 
 default:
-	@echo "You need to specify a subcommand."
+	@echo "You need to specify a subcommand. Type 'make help' for help."
 	@exit 1
 
 help:
-	@echo "build         - build docker containers for dev"
-	@echo "shell         - open a shell in the base container"
-	@echo "test          - run tests"
+	@echo "build        - install Python libs and build Docker containers"
+	@echo "test         - run tests"
+	@echo "testshell    - open a shell in the test container"
+	@echo "clean        - remove build files"
 
-# Dev configuration steps
-.docker-build:
-	make build
+.container-test: docker/test/Dockerfile requirements-dev.txt
+	${DC} build test
+	touch .container-test
 
-build:
-	${DC} build lambda
-	touch .docker-build
+build-containers: .container-test
 
-shell: .docker-build
-	${DC} run lambda bash
+build-libs:
+	${DC} run -u "${HOSTUSER}" test ./bin/run_build.sh
```

Comment:
I do this because then the dev doesn't need Python 3.6 running on their host computer. The test image has Python 3.6, so I figure we can use it as a "utility knife".